pkg/controllers/provisioning/v1alpha1/allocation/suite_test.go

```diff
@@ -39,7 +39,7 @@ import (
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 	RunSpecsWithDefaultAndCustomReporters(t,
-		"Provisioner",
+		"Provisioner-Allocator",
```

Comment:
Hmm, WDYT about Provisioner/Allocation? In general, I like the idea of parity between package name and suite name. After all, this is the suite for this package.

Reply:
I like the /, it keeps in line with other distinctions in annotation that we've done. I'll make that change.